src/ResourceManager/PowerBIEmbedded/ChangeLog.md

```diff
@@ -19,36 +19,53 @@
 -->
 ## Current Release
 
-## Version 4.0.0
-* Add support for online help
-    - Run Get-Help with the -Online parameter to open the online help in your default Internet browser
-    
-## Version 3.4.1
-
-## Version 3.4.0
-
-## Version 3.3.1
-
-## Version 3.3.0
-
-## Version 3.2.1
-
-## Version 3.2.0
-
-## Version 3.1.0
-
-## Version 3.0.1
-
-## Version 3.0.0
-
-## Version 2.8.0
-
-## Version 2.7.0
-
-## Version 2.6.0
-
-## Version 2.5.0
-
-## Version 2.4.0
-
-## Version 2.3.0
\ No newline at end of file
+* Add support for Power BI Embedded Capacity cmdlets - 
+
+* New Cmdlet Get-AzureRmPowerBIEmbeddedCapacity
+      Gets the details of a PowerBI Embedded Capacity.
+* New Cmdlet New-AzureRmPowerBIEmbeddedCapacity
+    - Creates a new PowerBI Embedded Capacity
+* New Cmdlet Remove-AzureRmPowerBIEmbeddedCapacity
+    - Deletes an instance of PowerBI Embedded Capacity
+* New Cmdlet Resume-AzureRmPowerBIEmbeddedCapacity
+    - Resumes an instance of PowerBI Embedded Capacity
+* New Cmdlet Suspend-AzureRmPowerBIEmbeddedCapacity
+    - Suspends an instance of PowerBI Embedded Capacity
+* New Cmdlet Test-AzureRmPowerBIEmbeddedCapacity
+    - Tests the existence of an instance of PowerBI Embedded Capacity
+* New Cmdlet Update-AzureRmPowerBIEmbeddedCapacity
+    - Modifies an instance of PowerBI Embedded Capacity
+
+-## Version 4.0.0
```

Comment:
@selasagi looks like these headers have a `-` character in front of them, which is affecting how the markdown is being rendered

Reply:
fixed